### PR TITLE
0.8.5 Release change.

### PR DESCRIPTION
**Pull request by rkooo567**
## Why are these changes needed?

This brings back some commits that I added to 0.8.5 release branch for a quick fix.  

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/latest/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failure rates at https://ray-travis-tracker.herokuapp.com/.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested (please justify below)

### CODE REVIEW

File: doc/dev/bin/organize_commit.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+"""
```
Comment (simon-mo) [inline doc/dev/bin/organize_commit.py:1]: move this to get_contributor? it doesn't make sense to add a different file

**Review by edoakes — APPROVED** [review]
Awesome, thanks for the improvements!

File: doc/dev/bin/ray-project/project.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+cluster:
+  config: ray-project/cluster.yaml
+
+commands:
+  - name: sanity-check
+    help: "Do release sanity check with pip"
+    command: |      
+      export RAY_VERSION={{ray_version}}
+      export RAY_HASH={{commit}}
+      chmod 555 ./pip_download_test.sh
```
Comment (simon-mo) [inline doc/dev/bin/ray-project/project.yaml:18]: chmod +x

File: doc/dev/bin/ray-project/project.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+# Pathnames for files and directories that should be saved
+# in a snapshot but that should not be synced with a# session. Pathnames can be relative to the project
+# directory or absolute. Generally, this should be files
+# that were created by an active session, such as
+# application checkpoints and logs.
+output_files: [
+  # For example, uncomment this to save the logs from the
+  # last ray job.
+  # "/tmp/ray/session_latest",
+]
```
Comment (simon-mo) [inline doc/dev/bin/ray-project/project.yaml:36]: newline

File: doc/dev/bin/sanity_check.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+import sys
+
+RAY_VERSION = "RAY_VERSION"
+RAY_COMMIT = "RAY_HASH"
+
+ray_version = os.getenv(RAY_VERSION)
+ray_commit = os.getenv(RAY_COMMIT)
+
+if __name__ == "__main__":
+    raise Exception
```
Comment (simon-mo) [inline doc/dev/bin/sanity_check.py:12]: ?

File: doc/dev/bin/sanity_check.py [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    print("Sanity check python version: {}".format(sys.version))
```
Comment (simon-mo) [inline doc/dev/bin/sanity_check.py:13]: also why can't we just run test_mini here?

File: doc/dev/microbenchmark_analysis.py [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+Usage: python microbenchmark_analysis.py
+"""
+
+from collections import defaultdict
+
+FRIST_VERSION = 0
+LAST_VERSION = 5
+FILES = [
+    "release_logs/0.8.{}/microbenchmark.txt".format(num)
```
Comment (simon-mo) [inline doc/dev/microbenchmark_analysis.py:14]: use glob so you don't need to change the LAST_VERSION each release

**Review by simon-mo — APPROVED** [review]
Looks good. Thanks for the change!

**Comment by rkooo567** [comment]
@simon-mo Addressed all comments. Please merge it.

**Pull request merged**